tests/ut/ir/transforms/test_submit_end_to_end.py

```
@@ -0,0 +1,98 @@
+            scratch: pl.Out[pl.Tensor[[16, 256], pl.FP32]],
+        ) -> pl.Tensor[[16, 256], pl.FP32]:
+            return scratch
+
+        @pl.function(type=pl.FunctionType.Orchestration)
+        def main(
+            self,
+            x: pl.Tensor[[16, 256], pl.FP32],
+            out: pl.Out[pl.Tensor[[16, 256], pl.FP32]],
+        ) -> pl.Tensor[[16, 256], pl.FP32]:
+            with pl.manual_scope():
+                scratch, tid = pl.submit(self.stage1, x, out)  # noqa: F841
```

Ruff (0.15.14)

[warning] Unpacked variable `scratch` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)